### PR TITLE
MM-40853 : Pressing Ctrl/Cmd+/ on MacOS should toggle Keyboard Shortcuts modal

### PR DESCRIPTION
**Pull request by VishakhaPoonia**
#### Summary
Pressing Ctrl+/ (or Cmd+/ if you're on Mac) to open the Keyboard shortcuts modal open's modal but hitting it back doesn’t close it this pr will fix that.

#### Ticket Link
  Fixes https://github.com/mattermost/mattermost-server/issues/19520
  Fixes https://mattermost.atlassian.net/browse/MM-40853

#### Related Pull Requests
``NONE``

#### Release Note
```release-note
Bug fix on closing keyboard shortcut modal by "ctrl/cmd + /" when its open
```

### CODE REVIEW

**Comment by VishakhaPoonia** [comment]
@hmhealey Could you please let me know the further process.

**Comment by M-ZubairAhmed** [comment]
@VishakhaPoonia thank you for sending in the PR, we will start the review soon. Apologies for the delay from our side

**Review by AshishDhama — APPROVED** [review]

**Comment by AshishDhama** [comment]
LGTM!

**Comment by AshishDhama** [comment]
Checked working as expected, will defer QA as changes are less than 50 lines

File: components/sidebar/index.ts
Comment (M-ZubairAhmed) [inline components/sidebar/index.ts:24]: Absolute path are preferred over relative path import.

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
It looks good just a minor suggestion.
Also would you be open to writing e2e test for this one? if you need help with that please drop by [Developers channel on community](https://community-daily.mattermost.com/core/channels/developers)

**Comment by VishakhaPoonia** [comment]
@M-ZubairAhmed Have added your suggestion, I am not very familiar with E2E but yeah would love to write, should that be part of this pr. As I mostly work on weekends so it might take some time.

**Comment by M-ZubairAhmed** [comment]
@VishakhaPoonia actually that can be a separate ticket. @jgilliam17 could you please check if we have a zephyr test for that. If not can you please create one. I will open up a ticket later. And @VishakhaPoonia, if you have time around weekends feel free to work on it. no pressure  :)

**Comment by VishakhaPoonia** [comment]
@M-ZubairAhmed If you create new ticket for E2E could you please assign that to me.

**Comment by M-ZubairAhmed** [comment]
> @M-ZubairAhmed If you create new ticket for E2E could you please assign that to me.

Sure will do that. I will wait for confirmation from @jgilliam17 :)

**Comment by M-ZubairAhmed** [comment]
@VishakhaPoonia can you please try to update the pr to resolve the conflicts.
@jgilliam17 gentle ping to review :)

**Comment by jgilliam17** [comment]
Thanks @VishakhaPoonia 
Tested, looks good. Will update and re-run the e2es before merging.
- Verified cmd+/ opens and closes the keyboard shortcuts modal
@M-ZubairAhmed We have an automated test for this keyboard shortcut, currently unstable. I will follow up with Saturn to see if the change I suggested is enough to promote the test to production.

**Comment by jgilliam17** [comment]
@VishakhaPoonia can you please update manually. Thanks

**Comment by VishakhaPoonia** [comment]
@jgilliam17 yeah, had to give my laptop for repairs got it now will update it today

**Comment by VishakhaPoonia** [comment]
@M-ZubairAhmed @jgilliam17 I have updated the branch, please take a look and let me know.

**Comment by jgilliam17** [comment]
Thanks @VishakhaPoonia 
Tested, looks good to merge.
[E2E report](https://mattermost-cypress-report.s3.amazonaws.com/158234-944389e-onprem-ent-webapp-pr-9802-mattermost/mm-ee-test944389e/mochawesome.html) looks good, no PR related failures. This is the run from March 3rd, the one from today failed to start, probably a cypress issue.

**Review by jgilliam17 — APPROVED** [review]

**Comment by jgilliam17** [comment]
@M-ZubairAhmed I didn't merge yet. Here is the link to [test case ](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/4064362) if you wanted to take a look. We can add here and I can re-test if any changes are made. Let me know.

**Comment by M-ZubairAhmed** [comment]
@jgilliam17 That test case is included in our e2e https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_2_spec.js#L41
so i dont think @VishakhaPoonia we need to add e2e for this pr. Once again thank you so much for working on it.

**Pull request merged**
